core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1045,6 +1045,28 @@ write_files:
             initialDelaySeconds: 15
             timeoutSeconds: 15
 
+  - path: /etc/kubernetes/manifests/kube-rescheduler.yaml
+    content: |
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        name: rescheduler-v0.2.2
+        namespace: kube-system
+        labels:
+          k8s-app: rescheduler
+          version: v0.2.2
+          kubernetes.io/cluster-service: "true"
+          kubernetes.io/name: "Rescheduler"
+      spec:
+        hostNetwork: true
+        containers:
+        - name: rescheduler
+          image: gcr.io/google-containers/rescheduler:v0.2.2
+          # TODO: Make resource requirements depend on the size of the cluster
```

Comment:
So, do we need to manually modify `/etc/kubernetes/manifests/kube-rescheduler.yaml` to update the `resources.requests` after each time worker nodes are scaled out considerably?
Better to document it?